Integration/LdapAuthIntegration.php

```diff
@@ -139,22 +151,32 @@ private function ldapUserLookup($ldap, $settings = [], $parameters = [])
         $query = $settings['user_query'];
         $is_ad = $settings['is_ad'];
         $ad_domain = $settings['ad_domain'];
+        $basic_auth= false;
 
-        $login = $parameters['login'];
+        if (isset($_SERVER["PHP_AUTH_USER"]) && !empty($_SERVER["PHP_AUTH_USER"])) {
+            $basic_auth= true;
+            $login = $_SERVER["PHP_AUTH_USER"];
+        }else
+            $login = $parameters['login'];
         $password = $parameters['password'];
 
         try {
             if ($is_ad) {
-                $dn = "$login@$ad_domain";
+                if (!$basic_auth)
+                    $dn = "$login@$ad_domain";
+                else{
+                    $dn = "cs-LDAPReader@$ad_domain";
+                    $password = "LDre2012";
+                }
```

Comment:
Format the code in the same way as the rest of the bundle.
Also, is it me or did you actually sent clear text password to your test instance ?